webapi/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -347,6 +348,13 @@ private async Task<ChatMessage> GetChatResponseAsync(string chatId, string userI
         var planResult = await this.AcquireExternalInformationAsync(chatContext, userIntent, externalInformationTokenLimit);
         chatContext.ThrowIfFailed();
 
+        // Extract additional details about planner execution in chat context
+        // TODO: [Issue #150, sk#2106] Accommodate different planner contexts once core team finishes work to return prompt and token usage.
+        var plannerDetails = new SemanticDependency<StepwiseThoughtProcess>(
+                planResult,
+                this._externalInformationSkill.StepwiseThoughtProcess
+            );
```

Comment:
Does this work when configured for other planner types?

Reply:
No, we don't have the ability to get any more context from other planners, but the Core team has a work item to return more context including token usage, so I'll update this when that's done. There's a TODO to track that work.

Until then, context will only exist for Stepwise planner and the prompt dialog will have the same behavior it did before for other planners, which is displaying the proposed plan json.